src/routes/search_routes.js

```diff
@@ -297,7 +299,7 @@ const searchCubes = async (query, order, lastKey, ascending, user) => {
   // separate query into tokens, respecting quotes
   const tokens = tokenize(query);
 
-  const cardQueries = getCardQueries(tokens);
+  const cardQueries = getCardQueries(tokens, user?.defaultPrinting);
```

Comment:
Even though we hydrate defaultPrinting in User, since search and tools can be used anonymously we still must use `?.` to handle no user